CHANGELOG.md

```diff
@@ -18,6 +18,12 @@ The public API of this library consists of the functions declared in file
 ### Breaking changes
 - rename `maxH3ToChildrenSize` to `cellToChildrenSize`
 - `cellToChildrenSize` to output `int64_t` instead of `int`
+- rename `GeoPoint` to `LatLng`
```

Comment:
rename `GeoCoord` is the change, right? Similar for the others should be the 3.7.1 names, unless we release a 4.0.0 release candidate.

Reply:
Hmm, I guess we actually missed adding a changelog note for #403. I'll put that here, and that should cover things. Sound reasonable?